index.js

```diff
@@ -235,7 +232,7 @@ export default class Supercluster {
             const clusterProperties = reduce ? this._map(p, true) : null;
 
             // encode both zoom and point index on which the cluster originated
-            const id = (i << 5) + (zoom + 1);
+            const id = (i << 5) + (zoom + 1) + this.points.length;
```

Comment:
If we're changing the meaning of cluster `id`, we should also change how it's decoded in methods like `getLeaves`, `getChildren` and `getClusterExpansionZoom`, otherwise those will break.

Reply:
ok will change now

Reply:
`getLeaves` decodes it through `getChildren` so should be good 👍